resources/integration_test/testdata/table_ops.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
Can you add a few negative ints and numbers in here. It should not matter and the folded query should still be the same (only the result shoud change) but it wouldn't hurt.